uds/can/transport_interface/common.py

```
@@ -391,15 +387,11 @@ def n_br(self, value: TimeMillisecondsAlias) -> None:
-        :raise TypeError: Provided value is not int or float type.
-        :raise ValueError: Provided value is out of range (0 <= value < MAX N_Br).
         """
-        if not isinstance(value, (int, float)):
-            raise TypeError(f"Provided time parameter value must be int or float type. Actual type: {type(value)}.")
-        if not 0 <= value < self.n_br_max:
-            raise ValueError("Provided time parameter value is out of range. "
-                             f"Expected: 0 <= value < {self.n_br_max}. Actual value: {value}.")
+        validate_time(value, accept_zero=True)
+        if value >= self.n_br_max:
+            raise ValueError("Provided time parameter value is greater than N_Br Max value. "
+                             f"Expected: value < {self.n_br_max}. Actual value: {value}.")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)